js/RegParams.js

```diff
@@ -17,15 +17,15 @@ export function calculateRegressionParams () {
     const meanOfY = sumY/numberOfPointsOnCanvas;
 
     regressionPlot.data.datasets[0].data.forEach(ele => {
-        numerator += (ele.x - meanOfX)*(ele.y - meanOfY); // change it to a more meaningful variable name
-        denominator += (ele.x - meanOfX)*(ele.x - meanOfX); // change it to a more meaningful variable name
+        numerator += ele.x * (ele.y - meanOfY); // change it to a more meaningful variable name
```

Comment:
Can you paste the actual formula for this calculation here.

Reply:
Using the earlier formula, I was able to match the regression params values with ones given in the link:
https://www.graphpad.com/quickcalcs/linear2/
Can @sbh69840 explain the reason for making the above change?

Reply:
Okay, will paste the actual formula. Regarding @sanya5549 's comment, the formula was correct itself. I was facing some issue with plotter and I thought it was in calculating regression params, so I changed here. I resolved the plotter issue and in the end, thought this was lesser code, so kept it anyway.

Reply:
is this formula correct? or changes are required?

Reply:
will paste the formula when I make the changes.